packages/components/src/base-control/test/index.tsx

```diff
@@ -46,6 +46,7 @@ describe( 'BaseControl', () => {
 
 		expect( textarea ).toHaveAttribute( 'aria-details' );
 		expect(
+			// eslint-disable-next-line testing-library/no-node-access
```

Comment:
This will have to be intentionally ignored, unless RTL gets better support for `aria-details`.